ironfish-cli/src/commands/status.ts

```diff
@@ -103,11 +101,23 @@ function renderStatus(content: GetStatusResponse): string {
     workersStatus += ` - ${content.workers.queued} -> ${content.workers.executing} / ${content.workers.capacity} - ${content.workers.change} jobs Δ, ${content.workers.speed} jobs/s`
   }
 
+  const heapTotal = FileUtils.formatMemorySize(content.memory.heapTotal)
+  const heapUsed = FileUtils.formatMemorySize(content.memory.heapUsed)
+  const rss = FileUtils.formatMemorySize(content.memory.rss)
+
+  const memoryStatus = `${heapUsed} / ${heapTotal} (${(
```

Comment:
What do you think about adding a description for each section (similar to how we have peer data)?

```
Heap: 50.15 MiB / 70.61 MiB (71.0%), RSS: 669.36 MiB (1.0%), Free: 15.57 GiB (75.7%)
```

Reply:
done